src/pages/[platform]/build-a-backend/storage/copy-files/index.mdx

```diff
@@ -60,3 +64,51 @@ const copyFile = async () => {
 Cross identity ID copying is only allowed if the source object's access level is set to 'protected'.
 
 </Callout>
+
+### More `copy` options
```

Comment:
Same feedback as above. You can collapse this under one heading of "Use with S3 Transfer Acceleration"